provider/devices/tizen.go

```diff
@@ -0,0 +1,353 @@
+package devices
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"regexp"
+	"slices"
+	"strings"
+	"sync"
+	"time"
+
+	"GADS/common/cli"
+	"GADS/common/models"
+	"GADS/provider/logger"
+	"GADS/provider/providerutil"
+)
+
+type TizenTVInfo struct {
```

Comment:
Can you move the struct types to `common\models` :P

Reply:
Done in [commit](https://github.com/shamanec/GADS/pull/208/commits/0b4d8d41de2827ef82137035be41c3f528ba937c)